### PR TITLE
LD_PUBDEV-4563 updated the python api docs to in H2OAutoML documentation

### PR DESCRIPTION
**Pull request by laurendiperna**

### CODE REVIEW

**Comment by navdeep-G** [comment]
LGTM. Lets merge once tests pass. cc @angela0xdata @abal5 @ledell

File: h2o-py/h2o/automl/autoh2o.py [hunk 1]
```diff
@@ -13,10 +13,41 @@ class H2OAutoML(object):
...
     and a Stacked Ensemble of all the models.
 
+    :param int max_runtime_secs: This argument controls how long the AutoML run will execute. Defaults to 3600 seconds (1 hour).
+    :param int max_models: Specify the maximum number of models to build in an AutoML run. (Does not include the Stacked Ensemble model.)
+    :param str stopping_metric: Specifies the metric to use for early stopping. Defaults to ``"AUTO"``.
+      The available options are:
+      ``AUTO`` (This defaults to ``logloss`` for classification, ``deviance`` for regression),
+      ``deviance``, ``logloss``, ``mse``, ``rmse``, ``mae``, ``rmsle``, ``auc``, ``lift_top_group``,
+      ``misclassification``, ``mean_per_class_error``.
+    :param int stopping_rounds: This argument stops training new models in the AutoML run when the option selected for
```
Comment (navdeep-G) [inline h2o-py/h2o/automl/autoh2o.py]: Looks like you're missing stopping tolerance?

**Comment by navdeep-G** [comment]
Minor comment about stopping tolerance.

File: h2o-py/h2o/automl/autoh2o.py (hunk 1 above)
Comment (laurendiperna) [inline h2o-py/h2o/automl/autoh2o.py]: thanks added it in

**Comment by angela0xdata** [comment]
LGTM

**Pull request merged**